docs/original_alexandria_rfc.md

```diff
@@ -0,0 +1,66 @@
+## Rationale
+Documents can currently only be added to a form via file question. These do not provide search, tags, thumbnails and some other features needed for a document management service. File questions can only be used inside a form.
```

Comment:
I'd rephrase this slightly to reference caluma. Might not be obvious to "outsiders".

```suggestion
Documents in Caluma can currently only be added to a form via file question. These do not provide search, tags, thumbnails and some other features needed for a document management service. File questions can only be used inside a form.
```